uproot_methods/classes/TGraph.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2018, DIANA-HEP
```

Comment:
Matthieu, why do you use this copyright in new files given that you are not working for the DIANA project?

Reply:
It was copy/paste from the other class definition files. I didn't know If I should put the copyright or not since they are in most of the files I have seen in the package.